code/game/machinery/crusher_piston.dm

```diff
@@ -79,6 +77,38 @@
 	// Change the icons of the neighboring bases
 	change_neighbor_base_icons()
 
+/obj/machinery/crusher_base/Destroy()
+	qdel(pstn)
+	return ..()
+
+/obj/machinery/crusher_base/attackby(var/obj/item/O as obj, var/mob/user as mob)
+	if(default_deconstruction_screwdriver(user, O))
+		return
+	//TODO: Add a chance to catch their hand in the mechanic if they do something while the crusher is in operation
+	if(default_deconstruction_crowbar(user, O))
+		return
+	if(default_part_replacement(user, O))
+		return
+	
+	//Stuff you can do if the maint hatch is open
+	if(panel_open)
+		if(iswrench(O))
+			valve_open = !valve_open
+			user << "<span class='notice'>You [valve_open ? "open" : "close"] the de-preassure valve of [src].</span>"
```

Comment:
Pressure. There's probably a better way to put this as well; "relief valve", maybe?